Procfile

```diff
@@ -1,2 +1,3 @@
 worker: bundle exec sidekiq -C ./config/sidekiq.yml
 publishing-queue-listener: bundle exec rake message_queue:listen_to_publishing_queue
+govuk-index-queue-worker: bundle exec rake message_queue:insert_data_into_govuk
```

Comment:
I think I chose the name `-listener` rather that worker to differentiate it from a internal "worker". Might be worth keeping that convention (but I don't feel strongly about it).